decidim-core/app/cells/decidim/redesigned_profile_cell.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module Decidim
+  class RedesignedProfileCell < Decidim::ViewModel
+    include Decidim::ApplicationHelper
+    include Decidim::Core::Engine.routes.url_helpers
+    include Decidim::UserProfileHelper
+    include Decidim::AriaSelectedLinkToHelper
+    include Decidim::LayoutHelper
+    include ActiveLinkTo
+
+    delegate :current_organization, :current_user, :user_groups_enabled?, to: :controller
+    delegate :avatar_url, :nickname, :personal_url, :followers_count, :public_users_followings, :following_count, to: :presented_profile
+
+    TABS_ITEMS = {
+      activity: { icon: "bubble-chart-line", path: :profile_activity_path },
+      badges: { icon: "award-line", path: :profile_badges_path },
+      following: { icon: "user-shared-line", path: :profile_following_path },
+      followers: { icon: "user-received-line", path: :profile_followers_path },
+      groups: { icon: "team-line", path: :profile_groups_path },
+      members: { icon: "contacts-line", path: :profile_members_path },
+      conversations: { icon: "question-answer-line", path: :profile_conversations_path }
+    }.freeze
+
+    def show
+      return render :invalid if profile_holder.blank?
+      return render :inaccessible if profile_holder.blocked? && current_user_logged_in?
+
+      render :show
+    end
+
+    private
+
+    def profile_holder
+      model
+    end
+
+    def presented_profile
+      present(profile_holder)
+    end
+
+    def show_badge?
+      return if user_group?
+
+      profile_holder.officialized?
+    end
+
+    def details_items
+      [{ icon: "account-pin-circle-line", text: nickname }].tap do |items|
+        items.append(icon: "link", text: personal_url) if personal_url.present?
```

Comment:
In case the personal URL is very long it breaks the view at least on mobile:
![Long personal URL](https://user-images.githubusercontent.com/864340/183034034-b3addd6f-b9e5-4af4-b1d8-0e21d9276a9d.png)

It also makes the mid and large sizes a bit funky looking but not as broken as mobile. Maybe we should wrap the text to make it break automatically to new lines?

Also note that although the default nickname length is configured at 20 characters, application administrators can change that, so that can be also long under some occasions (although most of the times it's kept pretty short).